docker/daemon/daemon_src.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/containers/image/manifest"
 	"github.com/containers/image/types"
 	"github.com/docker/docker/client"
-	"github.com/opencontainers/go-digest"
+	digest "github.com/opencontainers/go-digest"
```

Comment:
`go-digest` package should indeed contain the `digest` package and golang is able to resolve that. Tests run fine for me as well, not sure what you're doing wrong :confused: